cms/templates/import.html

```diff
@@ -102,8 +142,20 @@ <h3 class="title">${_("Verifying")}</h3>
               </span>
 
               <div class="status-detail">
-                <h3 class="title">${_("Updating Course")}</h3>
-                <p class="copy">${_("Integrating your imported content into this course. This may take a while with larger courses.")}</p>
+                <h3 class="title">
+                    %if library:
+                        ${_("Updating Library")}
+                    %else:
+                        ${_("Updating Course")}
+                    %endif
+                </h3>
+                <p class="copy">
+                    %if Library:
+                        ${_("Integrating your imported content into this library. This may take a while longer with larger libraries.")}
```

Comment:
@Kelketek Suggested replacement for the 2nd sentence: "This process might take longer with larger libraries." @marcotuts OK?